code/comic-book-binder.js

```diff
@@ -96,7 +96,7 @@ export class ComicBookBinder extends BookBinder {
 
         if (foundError) {
```

Comment:
Can you find out why you are getting into this if-branch?  i.e. why is foundError true for your comic book?

(Just log the e values to the console on line 89)

Reply:
I can share you for a short period. 

https://drive.google.com/open?id=1rm5RglMmhIkh1HIJa-mQImvQ-bR2w9YI

You will see that the problem is the naming of the file within the archive. As a result, the page order is wrong in kthoom

Regards

Olivier

Reply:
Thanks - I was able to grab the books and take a look.  You are right, the page order is incorrect due to the naming in the file.  Unfortunately, this is a common problem with comic book archive files and readers :-/

I do not get into the foundErr if-block, so I think your change is actually applied to the wrong line (it should be within the sort function).